addons/knowledge/static/src/models/activity/chatter.js

```diff
@@ -0,0 +1,15 @@
+/** @odoo-module **/
```

Comment:
PR/commit message - rewording a bit and fix typos : 
```
Allow user to find Knowledge articles with the command palette,
which can be opened with CTRL+K or anywhere where the palette can 
be opened.

Articles can be searched by typing "?" followed by the query string. 
The query string is used to match the name of the article and the 
article's parent name (content of the article is not used).

Once the user chose an article, them will be redirected to the selected
article in the knowledge app. Later commits will allow more
integrations between the knowledge app and chatter in the other apps,
to include, notably, some parts of an article in notes or messages on
other models.

A bridge between legacy and owl has been made in
web/static/src/legacy/utils.js since form views don't yet support owl.
Some demo data has been added for testing, to be changed later.

Task xxx
```

`Some demo data has been added for testing, to be changed later.` can be removed
and you can put those demo data in a separated commit.

Reply:
Ok so what about

```Allow user to find Knowledge articles with the command palette,
which can be opened with CTRL+K or anywhere the palette can 
be opened.

Articles can be searched by typing "?" followed by the query string. 
The query string is used to match the name of the article and the 
article's parent name (content of the article is not used).

Once the user chose an article, they will be redirected to the selected
article in the knowledge app. Later commits will allow more
integrations between the knowledge app and chatter in the other apps,
to include, notably, some parts of an article in notes or messages on
other models.

A bridge between legacy and owl has been made in
web/static/src/legacy/utils.js since form views don't yet support owl.

Task-2715436```

I restored all files that were lost duringthe rebase (including adding the chatter button), should I include them in this commit or split them up in multiple commits?